### PR TITLE
Update angular.js w/ npm auto-update

### PR DESCRIPTION
**Pull request by danmichaelo**
I missed angular.js 1.8.0 from cdnjs, so I made an attempt of configuring auto-update for it, using the instructions in [CONTRIBUTING.md](https://github.com/cdnjs/packages/blob/master/CONTRIBUTING.md#auto-update-config). Let's see if it passes CI. And let me know if you want the commit message on a different form.

### CODE REVIEW

File: packages/a/angular.js.json [hunk 1]
```diff
@@ -18,13 +17,16 @@
...
-    }
-  ]
+  "autoupdate": {
+    "source": "npm",
+    "target": "angular",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "angular.*"
```
Comment (drewfreyling) [inline packages/a/angular.js.json:27]: This won't grab `angular-csp.css`?

Comment (drewfreyling) [inline packages/a/angular.js.json:27]: We also probably don't want to grab `angular.min.js.gzip` either.

Comment (MattIPv4) [inline packages/a/angular.js.json:27]:
```suggestion
          "angular?(.min).js?(.map)"
```
Per @drewfreyling's comments, I think we just need the JS files + the map?

[Glob preview](https://www.digitalocean.com/community/tools/glob?comments=true&glob=angular%3F%28.min%29.js%3F%28.map%29&matches=false&tests=%2F%2F%20Imported%20from%20angular%20NPM%20package&tests=angular-csp.css&tests=angular.min.js.gzip&tests=angular.js&tests=angular.min.js&tests=index.js&tests=bower.json&tests=package.json&tests=angular.min.js.map&tests=LICENSE.md&tests=README.md)

Comment (danmichaelo) [reply packages/a/angular.js.json:27]: Interesting, I didn't check the fileMap since I assumed it was correct in the previous version.

Comment (danmichaelo) [reply packages/a/angular.js.json:27]: Right

Comment (danmichaelo) [reply packages/a/angular.js.json:27]: Thanks! But I think we also need `angular-csp.css`

**Comment by danmichaelo** [comment]
Changed it to `"angular*.@(js|css)?(.map)"` 

Test:  [Glob Tool](https://www.digitalocean.com/community/tools/glob?comments=true&glob=angular%2A.%40%28js%7Ccss%29%3F%28.map%29&matches=false&tests=README.md&tests=angular-csp.css&tests=angular.js&tests=angular.min.js&tests=angular.min.js.gzip&tests=angular.min.js.map&tests=bower.json&tests=index.js&tests=package.json)

**Pull request merged**